languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/hunspell/spelling.txt

```diff
@@ -151,3 +152,50 @@ Xbox
 Twitch
 iOS
 Alibaba
+concientizar
+UPS
+abuelita
+casita
+City
+Center
+River
+New
+mami
+papi
+Smith
+Hey
+AM
+PM
+Obama
+Nick
+Judith
+Miriam
+Paola
+Edwin
+Perez
+Tom
+Ivan
+Isaías
+Lucy
+Omar
+Luke
+Karla
+Jacob
+personita
+contáctanos
+desabasto
+pedacito
+Jones
+Gonzalo
+cajita
+cajitas
+EEUU
+EU
+AMLO
```

Comment:
disambiguation of new mexican persident: https://en.wikipedia.org/wiki/Andr%C3%A9s_Manuel_L%C3%B3pez_Obrador